taxi/forms.py

```diff
@@ -50,3 +50,35 @@ def validate_license_number(
         raise ValidationError("Last 5 characters should be digits")
 
     return license_number
+
+
+class SearchForm(forms.Form):
+    query = forms.CharField(
+        max_length=250,
+        required=False,
+        label="",
+        widget=forms.TextInput()
+    )
+
+    def __init__(self, *args, **kwargs):
+        placeholder = kwargs.pop("placeholder", "")
+        super(SearchForm, self).__init__(*args, **kwargs)
+        self.fields["query"].widget.attrs["placeholder"] = placeholder
+
+
+class CarSearchForm(SearchForm):
+    def __init__(self, *args, **kwargs):
+        kwargs["placeholder"] = "Search by model"
+        super(CarSearchForm, self).__init__(*args, **kwargs)
```

Comment:
You could simplify the code
```suggestion
        super().__init__(*args, **kwargs)
```